src/nvim/os/fs.c

```diff
@@ -202,6 +202,19 @@ int os_open(const char* path, int flags, int mode)
   return r;
 }
 
+/// Flushes file modifications to disk.
+///
+/// @param file_descriptor the file descriptor of the file to flush to disk.
+///
+/// @return `0` on success, a libuv error code on failure.
+int os_fsync(int file_descriptor)
```

Comment:
Yea but we use `file_descriptor` in other places in this file so I copied that. I can use `fd` if you want and maybe update the other one too?